Blocks/feature fix (#1503)
google/docsy: userguide/content/en/docs/adding-content/shortcodes/index.md

| ---------------- |------------| ------------|
| title | | The title to use.
| url | | The URL to link to.

> | url_text | The value of the `ui_read_more` parameter in the language file. For English, that's "Read more" | The link text to use.

Reviewer: ```suggestion
| url_text | The [language parameter](/docs/language/#internationalization-bundles) value of [`ui_read_more`](https://github.com/google/docsy/blob/main/i18n/en.toml)  followed by " …" | The link text to use.
```
